samples/routing/custom/Core_6/Billing/Program.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Threading.Tasks;
+using NServiceBus;
+
+class Program
+{
+
+    static void Main()
+    {
+        AsyncMain().GetAwaiter().GetResult();
+    }
+
+    static async Task AsyncMain()
+    {
+        Console.Title = "Samples.CustomRouting.Billing";
```

Comment:
I'm saying this because I know Mike has pointed it out. Might be easier to notice it's exactly the same thing, but on the other hand the difference is really tiny, so I'm ok with either.